### DISCUSSION TITLE
How to configure optional SSC?

### DISCUSSION THREAD

**Discussion by XTARZ** [Pryaxis/TShock#2564] [Q&A] [answered]
I created a private server using Tshock to play with friends, and I tried `sscconfig.json` enabled, but I expected a 'optional' ssc, which means one can join the server without login and have right access and can login to load a ssc.

Can I achieve that without plugins?

Currently I set ssc to true, and set guest group to `default`, but I still have to login to play normally

**Comment by bartico6** [comment, marked as answer]
And how would the server know which SSC file to load? :P

You can't make ssc optional without plugins, even if you solve the logistical problem above. By default it's all or none.

**Reply by XTARZ** [reply]
Got it. Thank you!
